### DISCUSSION TITLE
The 5000 Kingdoms, proposed for crown of kings.

### DISCUSSION THREAD

**Discussion by NanndoSnowzera** [sunflower-land/sunflower-land#1312] [Ideas]
**Hello everyone from the land of sunflowers, I'm here to bring you a draft proposal, I hope you like it. 
the proposal is basically for the crown of the goblins to become the crown of the kings of the 5000 kingdoms, and whoever owns it is entitled to build a small castle to buy the cheapest gold mining pickaxe (kings don't pay fees) but they need to of the tool right? nothing is free but wouldn't giving a discount to the kings a good idea?
in addition to giving the bonus of +15% in gold mining.
and also at the end of it all, it will help to burn a lot of resources since the castle is made of stone will be 500 Stones and 500 wood for all doors and materials inside the castle.
so here we see 2 positive points, the burning of resources on a large scale, and the appreciation of an asset in the long term for those who collect it, in addition to your NFT being able to wear the crown and look beautiful like a real king, right? haha ha
at the end of all you burn resource and for real the price, only for those who carry out burn actions within the game, actively mining and participating in the game for the long term. I believe this won't make a big difference in the game to the point of unbalance, for those who think in the long term it can help to evolve in the game and also visually not? valuing at least a little our NFT that we have so much love.**
_Illustrative photo below:_
![crown porposal LAND EXPANSION](https://user-images.githubusercontent.com/105903391/183310214-99041c48-4274-4271-92e4-b692d50e159a.jpg)
 EDIT 1 - **our friend mentioned in the comments about not limiting only to those who have the crowns, this would help to burn more resources and benefit everyone, so I replied with another addition in the proposal that everyone can build the castle and have the mining bonus but only the kings with crowns get the gold mining pickaxe discount. It's good for everyone to talk until there's a good and complete proposal.**

**Comment by rafaelcb-m** [comment]
Great idea

**Comment by katchaul** [comment]
Ideia muito boa no caso não servirá como uma verdadeira queima de recursos pois apenas 5000 pessoas participarão desse evento sendo que as lands foram esgotadas no caso 150k de pessoas no tanto 145k de pessoas ficarão de fora de evento, mas pode ser bom pois quem ficou ligado é gastou mais realmente vantagens tem e devem existir. Minha opinião mas essa ideia seria genial, no caso por que não existir duas Torres uma para quem tem a coroa e a outra obviamente menor e sem a coroa assim não deixaria pessoas de lado e aí sim seria uma verdadeira queima de estoque

**Comment by Yougottabekiddingme123** [comment]
The idea is ok but it should not be limited to owners of Goblin Crown. It should either be for everyone or for a new NFT that players can work towards.

**Reply by NanndoSnowzera** [reply]
how about everyone can buy the castle building, and get a 15% bonus in gold mining and whoever has the crown, gets the 15% bonus and also the 20% discount for kings when buying the pickaxe to mine gold. what did you think?

**Comment by katchaul** [comment]
it's just interesting

Em seg., 8 de ago. de 2022 às 00:19, NanndoSnowzera <
***@***.***> escreveu:

**Comment by Transvariando** [comment]
very interesting idea :O
